source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm

```diff
@@ -100,27 +116,26 @@ - (UIView *)renderButtons:(ACRView *)rootView
 
         [acoElem setElem:elem];
 
-        NSUInteger numElem = [childview subviewsCounts];
+        NSUInteger numElem = [[childview arrangedSubviews] count];
 
         UIButton *button = nil;
 
         @try {
             if ([acoElem meetsRequirements:featureReg] == NO) {
                 @throw [ACOFallbackException fallbackException];
             }
-            button = [actionRenderer renderButton:rootView inputs:inputs superview:superview baseActionElement:acoElem hostConfig:config];
+            button = [actionRenderer renderButton:rootView
+                                           inputs:inputs
+                                        superview:superview
+                                baseActionElement:acoElem
+                                       hostConfig:config];
             [childview addArrangedSubview:button];
         } @catch (ACOFallbackException *exception) {
-            handleActionFallbackException(exception,
-                                          superview,
-                                          rootView,
-                                          inputs,
-                                          acoElem,
-                                          config,
+            handleActionFallbackException(exception, superview, rootView, inputs, acoElem, config,
```

Comment:
if all the other changes were to change the parameters to be arranged vertically, why is this one to make the parameters arranged horizontally?

Reply:
this is done by clang-format according to Apple Coding style.

---
In reply to: [387950316](https://github.com/microsoft/AdaptiveCards/pull/3818#discussion_r387950316) [](ancestors = 387950316)